### ISSUE TITLE
rpcclient: Use btcutil.Address type in all interfaces

### ISSUE THREAD

**Issue by gracenoah** [btcsuite/btcd#1127]
Right now ImportAddress and some other functions take address as a string while ListUnspentMinMaxAddressesAsync and most other functions take it as btcutil.Address.

Everything function should take btcutil.Address in its interface for type safety.

See also 
* https://github.com/btcsuite/btcd/blob/fe786c93b6f4e0c96d6b19157b72963a9b50293d/rpcclient/wallet.go
* https://github.com/btcsuite/btcutil/blob/fd898ec77a5fdc8feac21487fe115fcdbc67cf7b/address.go#L105

**Comment by stevenroose** [comment]
I agree. I'll take this into account for my [gRPC implementation](https://github.com/btcsuite/btcd/pull/1075).
